### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by Nuri1977**
## Math magicians: events

1. Copied the files calculate.js and operate.js into a logic/ directory.
2. Imported the files in the Calculator component.
3. Implemented the event handlers to use the math logic.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,35 +1,58 @@
 import React, { Component } from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
 class Calculator extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
```
Comment (dasileker) [inline src/components/Calculator.js]: - [ ] Nice use of props but you forgot to validate them so please do.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi  @Nuri1977,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Nuri1977) [reply src/components/Calculator.js]: import PropTypes from 'prop-types';
Calculator.propTypes = { calculate: PropTypes.func.isRequired };

'calculate' PropType is defined but prop is never usedeslint[react/no-unused-prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/no-unused-prop-types.md)

If I do not use props, then I removed them.

**Comment by Nuri1977** [comment]
import PropTypes from 'prop-types';
Calculator.propTypes = { calculate: PropTypes.func.isRequired };

'calculate' PropType is defined but prop is never usedeslint[react/no-unused-prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/no-unused-prop-types.md)

If I do not use props, then I removed them.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Nuri1977,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**